### ISSUE TITLE
If DB connection is lost, site doesn't reconnect

### ISSUE THREAD

**Issue by massimocode** [SMH110/Pizza-website#244]

**Comment by massimocode** [comment]
```
Error handling request to /health/database { MongoError: Topology was destroyed 
at nextFunction (/app/back-end/node_modules/mongodb-core/lib/cursor.js:545:27) 
at Cursor.next [as _next] (/app/back-end/node_modules/mongodb-core/lib/cursor.js:699:3) 
at nextObject (/app/back-end/node_modules/mongodb/lib/cursor.js:680:8) 
at Cursor.next (/app/back-end/node_modules/mongodb/lib/cursor.js:270:12) 
at findOne (/app/back-end/node_modules/mongodb/lib/collection.js:1414:10) 
at Collection.findOne (/app/back-end/node_modules/mongodb/lib/collection.js:1400:44) 
at NativeCollection.(anonymous function) [as findOne] (/app/back-end/node_modules/mongoose/lib/drivers/node-mongodb-native/collection.js:130:28) 
at NodeCollection.findOne (/app/back-end/node_modules/mquery/lib/collection/node.js:42:19) 
at model.Query.Query.findOne (/app/back-end/node_modules/mquery/lib/mquery.js:1816:20) 
at model.Query.Query._findOne (/app/back-end/node_modules/mongoose/lib/query.js:1519:22) 
at /app/back-end/node_modules/kareem/index.js:250:8 
at /app/back-end/node_modules/kareem/index.js:23:7 
at _combinedTickCallback (internal/process/next_tick.js:131:7) 
at process._tickDomainCallback (internal/process/next_tick.js:218:9) name: 'MongoError', message: 'Topology was destroyed' } 
at=info method=GET path="/health/database" host= request_id=e9ec71af-5aba-4543-8eed-11d26d771a25 fwd="63.143.42.244" dyno=web.1 connect=0ms service=6ms status=500 bytes=370 protocol=http 
```

**Comment by massimocode** [comment]
Fixed in 3d7c5ab487a901b31c3ff2282792f969fba4af29
